cron.yaml

```diff
@@ -54,8 +54,4 @@ cron:
   url: /check-pull-request
   target: auto-submit
   schedule: every 1 minutes
-
-- description: check revert review issues for closed issues to update in our database
-  url: '/api/update-revert-issues'
-  target: auto-submit
-  schedule: every 24 hours
+  
```

Comment:
newline at EOF

Reply:
It is odd, I have the newline in my editor and when it gets sent here it does not show. Maybe github is not showing the newline on EOF?